app/assets/javascripts/discourse/app/components/create-invite-uploader.js

```diff
@@ -0,0 +1,112 @@
+import Component from "@ember/component";
+import getUrl from "discourse-common/lib/get-url";
+import discourseComputed from "discourse-common/utils/decorators";
+import {
+  displayErrorForUpload,
+  validateUploadedFiles,
+} from "discourse/lib/uploads";
+
+export default Component.extend({
```

Comment:
Looks like a lot of the code has been borrowed from the [uploads mixin](https://github.com/discourse/discourse/blob/master/app/assets/javascripts/discourse/app/mixins/upload.js). Why not import it like we do for every other `thing-uploader.js`?

Reply:
The upload mixin is a bit different. In this case, I wanted a more traditional form in a modal: there is some description, an input used to select the CSV file and then the submit button.

The upload mixin has a dropzone and will automatically upload anything that is dropped in there. It is not a good fit here.